File: 2.0/java/me/getBillingPlanUsage.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.account.model.BillingPlanUsage;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient callfireClient = new CallfireClient("api_login", "api_password");
        BillingPlanUsage billingPlanUsage = callfireClient.meApi().getBillingPlanUsage();
    }
}

File: 2.0/csharp/me/getBillingPlanUsage.cs
using CallfireApiClient;
using CallfireApiClient.Api.Account.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        BillingPlanUsage usage = client.MeApi.GetBillingPlanUsage();
    }
}
